lib/3_add_up_numbers.rb

```diff
@@ -5,5 +5,7 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def add_up_numbers(arr)
-  # Write your code here.
+ sum = 0
+arr.each { |x| sum+= x }
```

Comment:
Watch indentation here! Lines 8-10 should be in line with each other, all indented a space more than lines 7 and 11!